README.md

```diff
@@ -17,17 +12,31 @@ GET /:path
 
 ## Response
 
+An example response could look like this:
 ```json
 {
   "title": "Om Oss",
   "slug": "om-oss",
-  "updated_at": "2015-11-06T02:04:58Z",
-
-  "image": "unimplemented",
-
+  "updated_at": "2022-11-06T02:04:58Z",
+  "image": "https://example.com/static/hej.png",
+  "message": "hej",
   "body": "<h1>...",
   "sidebar": "<ul>...",
   "anchors": [{"id":"id", "value":"asdf"}],
+  "nav": [
+    {
+      "slug": "/om-oss",
+      "title": "Om Oss",
+      "sort": 1,
+      "active": true, 
+      "expanded": false
```

Comment:
`expanded` shouldn't show up when it's false (since it has `,omitempty` in the json tag thingie), right? But it would be awesome nice to see an example of what it looks like when a part of the nav is expanded!